src/_locales/en/messages.json

```diff
@@ -493,6 +493,10 @@
     "message": "New private window",
     "description": "New private window"
   },
+  "commandLabelFullscreen": {
```

Comment:
Please rename the command and all it's occurrences from `Fullscreen` to `EnterFullscreen` or "Enter full screen" respectively.